src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseRepository.cs

```diff
@@ -49,12 +49,24 @@ public class ReleaseRepository : IReleaseRepository
                 .Where(r => r.UserId == userId && r.Role != ReleaseRole.PrereleaseViewer)
                 .Select(r => r.ReleaseId)
                 .ToListAsync();
-            
+
+            var userPublicationIds = await _contentDbContext
+                .UserPublicationRoles
+                .Where(r => r.UserId == userId && r.Role == PublicationRole.Owner)
+                .Select(r => r.PublicationId)
+                .ToListAsync();
+            var userPublicationRoleReleaseIds = await _contentDbContext
+                .Releases
+                .Where(r => userPublicationIds.Contains(r.PublicationId))
+                .Select(r => r.Id)
+                .ToListAsync();
+            userReleaseIds.AddRange(userPublicationRoleReleaseIds);
```

Comment:
Am I being blind, or are there no tests for this?

Reply:
Yeah - my bad. For some reason I came in this morning and assumed I'd had already written all the tests necessary - I'll add test(s) for publication owners.